hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/lock/OMLockMetrics.java

```diff
@@ -203,5 +203,7 @@ public void getMetrics(MetricsCollector collector, boolean all) {
     MetricsRecordBuilder builder = collector.addRecord(SOURCE_NAME);
     readLockHeldTimeMsStat.snapshot(builder, all);
     readLockWaitingTimeMsStat.snapshot(builder, all);
+    writeLockHeldTimeMsStat.snapshot(builder, all);
+    writeLockWaitingTimeMsStat.snapshot(builder, all);
```

Comment:
Can we add a test case?. Probably you can refer: TestReplicationSupervisor
```

      MetricsCollectorImpl metricsCollector = new MetricsCollectorImpl();
      metrics.getMetrics(metricsCollector, true);
      Assert.assertEquals(1, metricsCollector.getRecords().size());
```

Reply:
Thank you for the suggestion! I have added a test case for this patch.